angular-app/src/app/app.module.ts

```diff
@@ -14,15 +15,18 @@ import { MatChipsModule } from '@angular/material/chips';
 import { MatDialogModule } from '@angular/material/dialog';
 import { MatDividerModule } from '@angular/material/divider';
 import { MatExpansionModule } from '@angular/material/expansion';
+import { MatFormFieldModule } from '@angular/material/form-field';
 import { MatIconModule } from '@angular/material/icon';
 import { MatInputModule } from '@angular/material/input';
+import { MatMenuModule } from '@angular/material/menu';
 import { MatListModule } from '@angular/material/list';
 import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
 import { MatSidenavModule } from '@angular/material/sidenav';
 import { MatToolbarModule } from '@angular/material/toolbar';
 import { MatTooltipModule } from '@angular/material/tooltip';
 import { CdkScrollableModule } from '@angular/cdk/scrolling';
 import { TextFieldModule } from '@angular/cdk/text-field';
+import { NgxAudioPlayerModule } from 'ngx-audio-player';
```

Comment:
Is this the same audio player or library being used by Jon? If not, is there a reason to use this player over the one Jon is using?

Reply:
I used a different one because I didn't think the post audio should be displayed as a waveform, but I'll ask Jon more specifically about how his library works and what it does.

Reply:
Yea, there may be a way to display post audio without the waveform. If so, i asy consolidate libraries. If it is not a simple option, what you have here is fine.

Reply:
After talking with Jon, he said that his library only shows the audio with the waveform so I'm going to keep it the same

Reply:
I feel like we could add a "moonshot" feature here so that the audio for posts can have a different visualization than the workspace audio.